src/collective/cover/tests/test_calendar_tile.py

```diff
@@ -10,10 +12,25 @@ class CalendarTileTestCase(TestTileMixin, unittest.TestCase):
 
     def setUp(self):
         super(CalendarTileTestCase, self).setUp()
+
+        self.request['year'] = 2016
+        self.request['month'] = 8
+
         self.tile = CalendarTile(self.cover, self.request)
         self.tile.__name__ = u'collective.cover.calendar'
         self.tile.id = u'test'
 
+        self.events = []
+        with api.env.adopt_roles(['Manager']):
+            for i in [3, 12, 21, 30]:
+                start = datetime(2016, 8, i, 10, 30)
+                end = datetime(2016, 8, i, 11, 30)
+                obj = api.content.create(
+                    container=self.portal, type='Event', id='e{0}'.format(i),
+                    startDate=start, endDate=end)
```

Comment:
@rodfersou this is not going to work as Dexterity-based content types field names are different than Archetypes-based.

see: https://github.com/plone/plone.app.event/issues/209